app/Http/Controllers/StationController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Station;
+use App\Services\StationService;
+use App\Http\Requests\StoreStation;
+use App\Http\Requests\UpdateStation;
+use Illuminate\Http\Response;
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+
+class StationController extends Controller
+{
+    public function __construct(StationService $station)
```

Comment:
Please take care of the details. Sometimes you're commenting methods and sometimes not - unify your approach. The same is for blank lines in your classes: sometimes there're lines before first method, sometimes there no lines between methods. Please read PSR-12 docs, including this chapter: https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements